### ISSUE TITLE
REST Reuest Step: Empty Request Window

### ISSUE THREAD

**Issue by karelhusa** [SmartBear/soapui#710]
I have test case for PetStore Sample  REST API. The test case includes a POST REST Request (to create a pet) and works fine.

![WorkingFine-2022-01-05_145242](https://user-images.githubusercontent.com/11407706/148228891-058b4f6b-b732-4a5b-b2f6-b83ee63ca912.jpg)

However, if I repeat the test step multiple times, the request window disappears. 

![EmptyRESTRequest2022-01-05_144030](https://user-images.githubusercontent.com/11407706/148228936-61fff36f-8fda-4646-b3bd-bfb4c3001967.jpg)

Then I can execute the test step and get the response. The response window is OK and is populated with a new response. 

![ResponseOK-2022-01-05_144426](https://user-images.githubusercontent.com/11407706/148228977-603feac5-ef59-498f-aa79-281cdec6d03c.jpg)

The request window remains empty. The Step Window cannot be closed (probably related to issue #616), The project cannot be closed either. I can only close SoapUI and restart it.

After restart, the test step works fine for some while, then the failure repeats.

**Comment by cryptton2004** [comment]
Yeah, happens a lot after closing the rest/http request window. The window itself doesn't close when you press the 'close' button, it just clears most of the content inside it and then hangs with just the border visible. Sometimes you can minimize/maximize the window again and some of the content will show (like the above screenshot with the 'Nothing is here') but most of the functionality will be unresponsive. The only workaround I've found is also restarting SoapUI.
No error is displayed in the SoapUI log, error log or any other log, but I haven't checked the actual log files generated by SoapUI. This is not specific to the latest version, it started occurring since 5.6
Some have reported it for previous version as well but I haven't personally encountered it below 5.6
The likeliness of someone fixing this issue is slim, however, so I'd suggest to just deal with it, since there's very little activity on the OS version and they just update it only if they have no other choice.
